controllers/mirrorpeer_controller.go

```diff
@@ -71,6 +71,7 @@ const spokeClusterRoleBindingName = "spoke-clusterrole-bindings"
 //+kubebuilder:rbac:groups=addon.open-cluster-management.io,resources=managedclusteraddons/status,verbs=*
 //+kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;create;update;watch
 //+kubebuilder:rbac:groups=console.openshift.io,resources=consoleplugins,verbs=get;list;create;update;watch
+// +kubebuilder:rbac:groups=view.open-cluster-management.io,resources=managedclusterviews,verbs=get;list;watch;create;update
```

Comment:
just a question: shouldn't this be added to `managedcluster_controller` file ?? instead of here...

Reply:
kubebuilder will add the required permissions for the pod to the role manifest automatically. All controllers run in the same container.

Reply:
yeah, but I assumed it was more of a practice to add RBACs on top of respective reconciler.

Reply:
I agree. It is good practice to have RBACs close to the controllers that require it. We haven't followed that well enough in this repo, but I think we should start doing it. End result is the same, but this makes it easy to argue about why a particular RBAC was added.